src/Tools/SnAdminRuntime/SnAdminRuntime.cs

```diff
@@ -75,11 +75,27 @@ internal static int Main(string[] args)
                 return -1;
 
             Logger.PackageName = Path.GetFileName(packagePath);
-
-            Logger.Create(logLevel, logFilePath);
-            Debug.WriteLine("##> " + Logger.Level);
-
-            return ExecutePhase(packagePath, targetDirectory, phase, parameters, logFilePath, help, schema);
+            try
+            {
+                Logger.Create(logLevel, logFilePath);
+                Debug.WriteLine("##> " + Logger.Level);
+                return ExecutePhase(packagePath, targetDirectory, phase, parameters, logFilePath, help, schema);
+            }
+            catch (System.Reflection.ReflectionTypeLoadException rtlex)
+            {
+                List<string> types = new List<string>();
+                foreach (var item in rtlex.LoaderExceptions) //LoaderExceptions is null? 
+                {
+                    if(item is System.IO.FileLoadException) //namespace-eket törüljük ki

```

Comment:
Please replace the current condition with an "as + null check" pair:
var fle = item as ....;
if (fle != null)
{...}